docs/ecosystem-roadmap.md

```diff
@@ -0,0 +1,101 @@
+
+### Terminal
+
+The terminal is the final layer providing graphical display and interactivity services to the user. It is responsible for capturing input and encoding it as [virtual terminal sequences](console-virtual-terminal-sequences.md) to eventually reach the client's `STDIN` and decoding and presenting the plain text and _virtual terminal sequences_ it received back from the client's `STDOUT`.
+
+#### Further Connections
+
+As an addendum, further connections can be performed by chaining applications that serve multiple roles into one of the endpoints. For instance, an SSH session has two roles: it is a Terminal for the command-line application running on one device, but it forwards all received information on to a Client role on another device. This chaining can occur indefinitely across devices and contexts offering broad scenario flexibility.
+
+On non-Windows platforms, the Server and Terminal roles are a single unit because there is no need for a translation compatibility layer between an API set and [virtual terminal sequences](console-virtual-terminal-sequences.md) as there is on Windows.
+
+## Products
+
+All of our products are now available on GitHub in our open source repository, [Microsoft/Terminal](https://github.com/microsoft/terminal).
+
+### Windows Console Host
+
+This is the traditional Windows user-interface for command-line applications. It handles all console API servicing called from any attached command-line application as well as the graphical user interface representation on behalf of all of those applications. It is found as `conhost.exe`, or `openconsole.exe` in its open source form. It comes with the Windows operating system and can be redistributed from the open source repository for a more up-to-date usage of the [pseudoconsole](pseudoconsoles.md) infrastructure with a modern terminal application. Per the definitions above, it operates in either a combined server and terminal role traditionally, or a server-only role through the preferred _pseudoconsole_ infrastructure.
+
+### Windows Terminal
+
+This is the new Windows interface for command-line applications, intended as a first-party example of using the [pseudoconsole](pseudoconsoles.md) to separate the concerns between API servicing and text-based application interfacing, much like all non-Windows platforms. It is intended to be the flagship text mode user interface for Windows and demonstrate the capabilities of the ecosystem, drive Windows development toward a unification with other platforms, and be an example of how to build a robust and complex modern application that spans the history and gamut of Windows APIs and frameworks. Per the definitions above, this product operates in a terminal role.
+
+## Roadmap

```

Comment:
Maybe "History & Roadmap"?

Reply:
I'd rather not, if that's OK. I think it's fine to talk about history and future as a map.